features/plugin-uninstall.feature

```diff
@@ -112,3 +164,5 @@ Feature: Uninstall a WordPress plugin
       """
     And the wp-content/languages/plugins/wordpress-importer-fr_FR.mo file should not exist
     And the wp-content/languages/plugins/wordpress-importer-fr_FR.po file should not exist
+    And the wp-content/languages/plugins/wordpress-importer-fr_FR.l10n.php file should not exist
```

Comment:
@swissspidy I added the test here for the php language file but its not clear how to me that I can test these things:

1. Test that if I have the `update_plugins` transient that any deleted plugins are removed
2. Test that if uninstall succeeds and delete fails that the user will get the error for that

I think those are the remaining logic branches to be tested but I don't see any code coverage reporting with Behar

Reply:
> Test that if I have the update_plugins transient that any deleted plugins are removed from it

You can run `wp transient get update_plugins` in the test and verify that the plugin is not part of the output.

> Test that if uninstall succeeds and delete fails that the user will get the error for that

You could for example check STDERR/STDOUT and whether the files still exist.

Reply:
> > Test that if I have the update_plugins transient that any deleted plugins are removed from it
> 
> You can run `wp transient get update_plugins` in the test and verify that the plugin is not part of the output.
> 

Thanks, I made a check for this


> > Test that if uninstall succeeds and delete fails that the user will get the error for that
> 
> You could for example check STDERR/STDOUT and whether the files still exist.

In this instance, I have no way to cause the `delete_plugin()` function to fail:

https://github.com/wp-cli/extension-command/blob/main/src/Plugin_Command.php#L1477

Reply:
Makes sense, I think it's ok not having a test for that last part.